fineract-provider/src/main/resources/application.properties

```diff
@@ -42,6 +42,8 @@ fineract.mode.write-enabled=${FINERACT_MODE_WRITE_ENABLED:true}
 fineract.mode.batch-worker-enabled=${FINERACT_MODE_BATCH_WORKER_ENABLED:true}
 fineract.mode.batch-manager-enabled=${FINERACT_MODE_BATCH_MANAGER_ENABLED:true}
 
+fineract.query.parameter-size=${FINERACT_QUERY_PARAMETER_SIZE:65000}
```

Comment:
2 things, I'd name this specifically that this is the maximum amount of parameters within an IN clause, emphasize that please.
Second thing, I'd be way more careful with the default number here and go with like 10-20k but definitely not 65k.

The thing is, DBs are often really bad at executing queries with really big IN clauses.

Reply:
As a default i would go with 1000 maximum.

Reply:
changed it